source/openapi3/split.yaml

```
@@ -3209,7 +3209,7 @@ components:
           description: The email of the Contact (256 max. characters)
         phone:
           type: string
-          description: The phone number of the Contact (11 max. characters)
+          description: The phone of the Contact. Must be a valid New Zealand mobile number in [ITU-T E.164](https://en.wikipedia.org/wiki/E.123) or [national](https://en.wikipedia.org/wiki/Telephone_numbers_in_New_Zealand#Mobile_phones) format
```

Would it be more relevant to link to the E.164 page?
```suggestion
          description: The phone of the Contact. Must be a valid New Zealand mobile number in [ITU-T E.164](https://en.wikipedia.org/wiki/E.164) or [national](https://en.wikipedia.org/wiki/Telephone_numbers_in_New_Zealand#Mobile_phones) format
```

---

Woops, it definitely would be 😅